swarm/src/connection.rs

```diff
@@ -848,6 +920,51 @@ mod tests {
         assert_eq!(connection.handler.local_removed, vec![vec!["/foo"]]);
     }
 
+    #[test]
+    #[ignore]
+    fn repoll_with_active_protocols() {
+        fn run_benchmark(protcol_count: usize, iters: usize) {
```

Comment:
I'd like us to use [criterion](https://docs.rs/criterion/latest/criterion/) for micro-benchmarking. It is a complex topic and we shouldn't hand-roll this.

Reply:
Does `cargo bench` set a `cfg` of some kind like `cargo test` sets `#[cfg(test)]`? Then we can make the re-export - which is probably just a `pub use` conditional on that.

Alternatively, we can make up our own `cfg` (not feature!) so conditionally expose it.

Reply:
We also don't need to necessarily keep the benchmarks (could even be commented out).